build-support/functions/10-util.sh

```diff
@@ -959,7 +959,7 @@ function shasum_directory {
    return $ret
 }
 
- function ui_version {
+function ui_version {
```

Comment:
Can we remove this now that we got rid of  the old UI?

Reply:
This function ensures the version is correct in the index.html of the new UI so it still needs to stay. We use it to check and make sure that we actually updated that version so we dont release consul 1.5.0 with a UI saying its 1.4.4 for example.